code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -683,19 +680,48 @@
 /datum/reagent/consumable/maltodextrin
 	name = "Maltodextrin"
 	description = "A common filler found in processed foods. Foods containing it will leave you feeling full for a much shorter time."
-	color = "#ffffff"
+	color = "#d4e1ee"
 	chem_flags = CHEMICAL_RNG_GENERAL
 	taste_mult = 0.1 // Taste the salt and sugar not the cheap carbs
 	taste_description = "processed goodness"
 	nutriment_factor = 0
 	metabolization_rate = 0.05 * REAGENTS_METABOLISM //Each unit will last 50 ticks
+	overdose_threshold = 5
 
 /datum/reagent/consumable/maltodextrin/on_mob_life(mob/living/carbon/M)
 	M.adjust_nutrition(-0.3) //Each unit will match nutriment 1:1 when completely processed
 	..()
 
-////Lavaland Flora Reagents////
+/datum/reagent/consumable/maltodextrin/overdose_start(mob/living/M)
+	if(!HAS_TRAIT(M, TRAIT_VORACIOUS))
+		to_chat(M, "<span class='notice'>Something in your guts feels off...</span>")
+		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "[type]_overdose", /datum/mood_event/guts_sickness, name)
+	else
+		to_chat(M, "<span class='notice'>Maybe I should have more junk foods today...</span>")
```

Comment:
```suggestion
```
There doesn't need to be a special message for voracious 

Also the massive YOU FEEL LIKE YOU HAD TOO MUCH OF message will completely overshadow the one listed above for non-voracious players.